Warning regarding HW14 and higher on 4.7.x failing (#30951)
openshift/openshift-docs: modules/installation-vsphere-infrastructure.adoc

====
You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
====

> 
> [IMPORTANT]
> ====
> VMs configured to use virtual hardware version 14 or greater may result in a failed installation.  It is recommended that virtual machines be configured with virtual hardware version 13.  This is a known issue that is being addressed in link:https://bugzilla.redhat.com/show_bug.cgi?id=1935539[BZ#1935539].

Reviewer: ```suggestion

[IMPORTANT]
====
Virtual machines (VMs) configured to use virtual hardware version 14 or greater might result in a failed installation. It is recommended to configure VMs with virtual hardware version 13. This is a known issue that is being addressed in link:https://bugzilla.redhat.com/show_bug.cgi?id=1935539[BZ#1935539].
```

Author: No problem at all, happy to help.  I just made those pushed those updates.
